DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/RefreshUIDProcessor.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
+{
+    public class RefreshUIDProcessor : IAnonymizerProcessor
```

Comment:
>RefreshUIDProcessor [](http://example.com/codeflow?start=17&length=19)

I guess you missed a constructor for RefreshUIDProcessor , right? Otherwise the replacedUIDs has no values...

Reply:
I see, you are feeding the dict by using processing... weird:confused:

Reply:
you really need full tests for your code...

Reply:
It's not weird but natural :smile: 
RefreshUID means creating a new UID for the item. the dictionary is a cache to make sure same original ID is replaced with the same new ID.
I guess.

Reply:
Yes, it used like cache. I modified it as static variable.